test/end-to-end/pages/declaration/declaration.js

```diff
@@ -1,14 +1,22 @@
 'use strict';
 
 const config = require('config');
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/declaration');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
+const contentEn = require('app/resources/en/translation/declaration');
+const contentCy = require('app/resources/cy/translation/declaration');
+const pagePath = require('app/steps/ui/declaration');
 
-module.exports = async function(bilingualGOP) {
+module.exports = async function(language = 'en', bilingualGOP) {
     const I = this;
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
+    const declarationContent = language === 'en' ? contentEn : contentCy;
+
+    await I.seeInCurrentUrl(pagePath.getUrl());
+    if (language === 'en') {
+        await I.waitForText(declarationContent.highCourtHeader, config.TestWaitForTextToAppear);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages